### DISCUSSION TITLE
如果向lm提出自定义主页支持套娃网页会被打吗（？

### DISCUSSION THREAD

**Discussion by Joker2184** [Meloong-Git/PCL#4646] [讨论]
想搞点不一样的（但是PCL太局限了（（（（

**Comment by tangge233** [comment]
WPF 支持网页组件的，但是很鸡肋。

**Reply by Joker2184** [reply]
已测试 不能用
具体参考微软WPF教程

**Reply by tangge233** [reply]
> 已测试 不能用
> 具体参考微软WPF教程

看不到内容罢了，其实是加载了
利用这个可以做一个背景音乐？

**Comment by hejiehao** [comment]
https://github.com/Hex-Dragon/PCL2/issues/2986

**Reply by Joker2184** [reply]
文内链接已失效（（
